clang/test/Frontend/sycl-propagate-aspect-warning.cpp

```diff
@@ -0,0 +1,97 @@
+// RUN: %clang_cc1 -internal-isystem %S/Inputs -fsycl-is-device -mconstructor-aliases -no-opaque-pointers -emit-codegen-only -verify %s
```

Comment:
Can we also check with early optimizations enabled?
Is `-mconstructor-aliases` necessary here?

Reply:
Found some information about this - https://stackoverflow.com/questions/6921295/dual-emission-of-constructor-symbols